index.html

```diff
@@ -344,6 +394,5 @@ <h2 class="contact">Contact me</h2>
         <button type="submit" class="btn btn-width">Get In Touch</button>
       </form>
     </section>
-  </div>  
-</body>
+  </body>
```

Comment:
- [ ] like the previous reviewer noted, kindly align the get in touch button with your input fields for the mobile version. Please use the `dimensions: responsive` to check it .